src/orders-v2/underline-link.tsx

```diff
@@ -1,23 +1,20 @@
 import { Text, TextProps } from "@saleor/macaw-ui-next";
 import React from "react";
-import { Link } from "react-router-dom";
+import { Link, LinkProps } from "react-router-dom";
 
-interface Props extends TextProps {
-  href: string;
-  children: React.ReactNode;
+interface UnderlineLinkProps extends LinkProps {
+  textProps?: Omit<TextProps, "children">;
 }
 
 // TODO: move to MacawUI
```

Comment:
we should move UnderlineText, not UnderlineLink

Reply:
Fixed in https://github.com/saleor/saleor-dashboard/pull/5780/commits/c25beaeb9a721fc72308419fbd1d4a41fd7fb4e1